ext/native_test.go

```diff
@@ -678,9 +698,15 @@ func mustParseTime(t *testing.T, timestamp string) time.Time {
 	return out
 }
 
+type TestStructWithMultipleSameNames struct {
+	Name        string
+	custom_name string `cel:"Name"`
```

Comment:
This is an interesting collision case. Can you document this behavior on the native.go library comments?